test-common/src/test/scala/org/scalajs/testcommon/SerializerTest.scala

```diff
@@ -28,4 +30,12 @@ class SerializerTest {
     assertEquals(in.toString(), out.toString())
     assertEquals(in.getStackTrace().size, out.getStackTrace().size)
   }
+
+  @Test
+  def serializeAndDesirializeIfFileNameIsNull: Unit = {
```

Comment:
Two things:
- Name: `serializeStackTraceElementWithNullFilename` (mainly for consistency with the other test).
- Reference #3611 (just add `// # 3611`).

Reply:
Addressed in https://github.com/scala-js/scala-js/pull/3631/commits/8cc42399ebe2655f0b7a45bcdb7415eb734b920f and https://github.com/scala-js/scala-js/pull/3631/commits/169adb6c0544a9e0587d096bdd07abc1d6eaf87a